recipes/astra-toolbox/bld.bat

```diff
@@ -0,0 +1,33 @@
+@echo off
+
+set R=%SRC_DIR%
+
+set B_VC=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\
+rem call "%B_VC%\vcvars64.bat" amd64 8.1 -vcvars_ver=14.0
```

Comment:
```suggestion
```

Reply:
Deleted these lines, they don't seem necessary in the conda-forge environment. They are from the source conda recipe.